build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -55,3 +40,45 @@ apiServer:
 
   # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
   #tlsMinVersion:
+
+# flowCollectorSink contains external IPFIX or JSON collector related configuration options.
+flowCollectorSink:
```

Comment:
I want to make sure @salv-orlando @jianjuns and @tnqn are also ok with the new config format

Reply:
Grouping parameters together looks good to me. I wonder why we add the "Sink" suffix - is it a convention for parameter groups?

Reply:
"Sink" is more of a convention for a place where data is being sent. It's the opposite of a data "source". https://en.wikipedia.org/wiki/Sink_(computing). Let us know if you think this is not an appropriate name.

Reply:
Oh, ok. "Collector" and "Sink" seems duplication then? Could we just name them `flowCollector` and `clickHouse`?

Reply:
either way works for me. @wsquan171

Reply:
I must admit I not only do not have a strong opinion, but in this case I do not have an opinion at all. Both names work for me.

Reply:
changed to `flowCollector` and `clickHouse` according to comment by @jianjuns